<!DOCTYPE html>
<!--[if IE 7 ]>    <html class="ie7 oldie"> <![endif]-->
<!--[if IE 8 ]>    <html class="ie8 oldie"> <![endif]-->
<!--[if IE 9 ]>    <html class="ie9"> <![endif]-->
<!--[if (gt IE 9)|!(IE)]><!--> <html> <!--<![endif]-->

<head>

    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"/>
    <meta charset="utf-8"/>
    <meta name="description" content="David Ochoa's Personal Website">
    <meta name="author" content="David Ochoa">
	<meta name="keywords" content="bioinformatics systems biology dochoa David Ochoa csbg mirrortree mtserver">
	<link rel= "icon" type="image/png" href="./favicon.png" />
    <title>David Ochoa's Personal Website</title>

    <link rel="stylesheet" href="css/style.css" type="text/css" media="screen" />
    <link rel="stylesheet" href="css/nivo-slider.css" type="text/css" />
    <link rel="stylesheet" href="css/jquery.fancybox-1.3.4.css" type="text/css" />

    <!--[if lt IE 9]>
	    <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->

    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.6/jquery.min.js"></script>
    <script>window.jQuery || document.write('<script src="js/jquery-1.6.1.min.js"><\/script>')</script>

    <script src="js/jquery.smoothscroll.js"></script>
    <script src="js/jquery.nivo.slider.pack.js"></script>
    <script src="js/jquery.easing-1.3.pack.js"></script>
    <script src="js/jquery.fancybox-1.3.4.pack.js"></script>
    <script src="js/init.js"></script>

	<script type="text/javascript">
		var _gaq = _gaq || [];
  		_gaq.push(['_setAccount', 'UA-38050760-1']);
  		_gaq.push(['_trackPageview']);
		(function() {
			var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
			ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
			var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
		})();
		</script>
</head>

<body>

<!-- header-wrap -->
<div id="header-wrap">
    <header>
        <hgroup>
            <h1><a href="index.html">David Ochoa</a></h1>
            <h3>Bioinformatics &amp; Systems Biology</h3>
        </hgroup>

        <nav>
            <ul>
				<li><a href="#main">Research</a></li>
				<li><a href="#publications">Publications</a></li>
                <li><a href="#others">Others</a></li>
                <li><a href="#contact">Contact</a></li>
            </ul>
        </nav>

    </header>
</div>

<!-- content-wrap -->
<div class="content-wrap">
	
	<section id="main" class="clearfix">
          <!-- <h1>Research Interests.</h1> -->
          <div class="primary">
              <p class="intro">I'm a Postdoctoral Fellow in <a href="http://www.evocellnet.com/p/research.html">Pedro Beltrao's group </a> at <a href="http://www.ebi.ac.uk">EMBL-EBI</a>. My research is focused on disentangling the complexity of biological systems at different levels. The understanding of this complexity plays a key role in the generation of data-driven hypothesis that can reveal the fundamental basis of disease, development, variability, evolution, etc. </p>
			  			
			<h2>Research interests</h2>
			<p>Genome-wide techniques are opening a new age in the study of biological systems. The generation of large amounts of data makes completely necessary the research in a new generation of concepts in order to process and analyze the incoming results. The integration of all this information is even a more ambitious task in which genomic, transcriptomic, or proteomic information - among others - could be combined to obtain more robust hypothesis.</p>
			
			<p>As part of my post-doctoral research, I'm particularly interested on the evolutionary dynamics, specificity and functional relevance of post-translational modifications (PTMs). Considering the growing proteomic resource available nowadays, the comparative analysis of genome-wide modification sites can provide more insight on the specificity and evolution of the regulatory mechanisms of protein activity.</p>
			
			<p>During my PhD in <a href="http://csbg.cnb.csic.es/">Florencio Pazos' group </a> at <a href="http://www.cnb.csic.es">CNB-CSIC</a>, my research was primary focused on improving computational methods for predicting protein-protein interactions based on coevolution. These methods are founded on the hypothesis that interacting or functionally related proteins tend to adapt to each other during the evolutionary process. On that way, we proposed different modifications to the standard methods in order to take advantage of the growing information on protein sequence data. Moreover, we developed a web server to allow non-expert users to easily explore the tree similarities in a taxonomic context.</p>			

          </div>
          <aside>
			  <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>
			  <a class="twitter-timeline" href="https://twitter.com/biodogcaesar" data-widget-id="345520546039087105">Tweets by @biodogcaesar</a>
          </aside>
         <a class="back-to-top" href="#top">Back to Top</a>    
	</section>
    
	<section id="publications" class="clearfix">
		<h1>Publications.</h1>
		<div class="primary">
			<h2>Journals</h2>
				<ul>
					<li><u>Ochoa, D</u>, García-Gutiérrez, P, Juan, D, Valencia, A &amp; Pazos, F: Incorporating information on predicted solvent accessibility to the co-evolution-based study of protein interactions. <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&id=23104128&retmode=ref&cmd=prlinks">Molecular bioSystems 2013, 9:70–76</a>.</li>
					<li>Herman D, <u>Ochoa D</u>, Juan D, Lopez D, Valencia A &amp; Pazos F: Selection of organisms for the co-evolution-based study of protein interactions. <a href="http://www.biomedcentral.com/1471-2105/12/363">BMC Bioinformatics 2011, 12:363</a>.</li>
					<li><u>Ochoa D</u> &amp; Pazos F: Studying the co-evolution of protein families with the Mirrortree web server.<a href="http://bioinformatics.oxfordjournals.org/content/26/10/1370.abstract">Bioinformatics 2010, 26:1370–1371</a>.</li>
				</ul>
			<h2>Events</h2>
			<ul>
				<li><u>Ochoa D</u>, García P, Juan D, Pazos F: Coevolution and Predicted Solvent Accessibility. <a href="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/posterlist.php?cat=L">ISMB/ECCB 2011 Vienna</a>.</li>
				<li><u>Ochoa D</u>, Pazos F: MirrorTree: a web server to study the coevolution between protein families. Jornadas Bioinformática 2009 Lisbon.</li>
			</ul>
		</div>
        <aside>
                <h2>Published Tools</h2>
                <ul class="link-list">
                    <li><a href="http://csbg.cnb.csic.es/mtserver" title="MirrorTree Server">MirrorTree Server</a></li>
                </ul>
        </aside>
		<a class="back-to-top" href="#top">Back to Top</a>
	</section>
	
	<section id="thesis" class="clearfix">
		<h1>Thesis.</h1>
		<div class="primary">
			<h3><a href="data/thesis/thesis.pdf">Improving Co-evolution Based Methods for Protein-Protein Interaction Prediction</a></h3>
			<p>As an alternative to complement the experimental methods, a set of computational approaches have tried to take advantage of the different evolutive landmarks that interacting proteins print on their genes. For instance, evidence suggests that functionally related and potentially interacting proteins tend to evolve in a coordinated way, thereby presenting similar phylogenetic trees. A particularly successful family of methods, known as <em>mirrortree</em>, has emerged to quantify this co-evolution at a sequence level as a sign of interaction at a molecular level. The main proposal of this thesis is to diagnose the problems limiting the full implementation of co-evolution-based methods, in order to offer possible solutions, potential applications and foreseeable developments. <a href="data/thesis/thesis.pdf">[pdf]</a> <a href="data/thesis/thesis_keynote.pdf">[defense]</a></p>
		</div>			
        <aside>
			<a href="data/thesis/thesis.pdf"><img class="align-right" alt="" src="images/thesis_cover.jpg" /></a>
		</aside>
        <a class="back-to-top" href="#top">Back to Top</a>			
	</section>
	
    <!-- services -->
    <section id="others" >
           <h1>Other activities.</h1>
          <div class="row">
              <section class="col">
                  <h2>Teaching</h2>
                  <p><img class="align-left" alt="" src="images/services/print-design.png" />Assistant Professor on <a href="http://www.masterbioinformatica.com">MsC on Bioinformatics and Computational Biology</a>. <a href="http://csbg.cnb.csic.es/master2012">Practical and theoretical lectures</a> on Biological Networks and Systems Biology from 2010-2012. </p>
              </section>
              <section class="col mid">
			      <h2>Reviewer</h2>
                  <p><img class="align-left" alt="" src="images/services/logo-design-and-branding.png" />Reviewer for Oxford University Press's Bioinformatics (Impact Factor: 5.468)</p>
	           </section>
              <section class="col">
	              <h2>Web Developer</h2>
                  <p><img class="align-left" alt="" src="images/services/webdevelopment.png" /></p>
					<p>Web Developer for several Websites including: <a href="http://sysbiol.cnb.csic.es/SysBiol/">Systems Biology Program</a>, <a href="http://csbg.cnb.csic.es/mtserver">MirrorTree Server<a/>, <a href="http://sysbiol.cnb.csic.es/vmeeting">Fostering Systems and Synthetic Biology in Southern Europe<a/>.
              </section>
          </div>
          <a class="back-to-top" href="#top">Back to Top</a>
    </section>

      <!-- contact -->
      <section id="contact" class="clearfix">
        <h1>Contact Me.</h1>  
		<div class="row">
              <section class="col">
	            <p>Please contact me for information about my profile, research, collaborations, postdoc positions, etc. </p>  
	          </section>
              <section class="col mid">
                  <p>
                  Computational Systems Biology Group <br />
                  Systems Biology Program <br />
                  c/ Darwin 3 (Campus Cantoblanco) <br />
				  28049 Madrid
                  </p>
                  <p>
                  Email: dochoa [@] cnb.csic.es
                  </p>
	           </section>
              <section class="col">
	              <h2>Follow Me</h2>
                  <ul class="link-list social">
                      <li class="twitter"><a href="http://twitter.com/biodogcaesar">Twitter</a></li>
                      <li class="linkedin"><a href="http://www.linkedin.com/pub/david-ochoa/21/529/b78">Linkedin</a></li>
                  </ul>
	          </section>
          </div>
	 </section>
</div>

<!-- footer -->
<footer>
    <div class="footer-content">
        <ul class="footer-menu">
			<li><a href="#main">Research</a></li>
			<li><a href="#publications">Publications</a></li>
            <li><a href="#others">Others</a></li>
            <li><a href="#contact">Contact</a></li>
        </ul>
        <p class="footer-text">Copyright 2012 David Ochoa. &nbsp;&nbsp;&nbsp; Modified from the original design of <a href="http://www.styleshout.com/">Styleshout</a></p>
    </div>

</footer>

</body>
</html>
