<html>
	<head>
		<meta charset="utf-8"/>
		<link href="../../CSBG.css" rel="stylesheet" type="text/css">
		<meta name="keywords" content="mirrortree accessibility prediction David Ochoa csbg mirrortree">
		<title>Incorporating information on predicted solvent accessibility to the co-evolution-based study of protein interactions - Computational Systems Biology Group </title>
	</head>
	
<body bgcolor="#FFFFFF"  onLoad="parent.BGPmain.location='home.html'">

<!--ESCRIBE AQUÃ EL CONTENIDO -->

<i><a href="http://csbg.cnb.csic.es/">[CSBG main page]</a></i>
<p class="sectionP">Incorporating information on predicted solvent accessibility to the co-evolution-based study of protein interactions </p>


<a href="Figure1.png"><img src="Figure1.png" align="right" width="350px"></a>

<p><a href="http://pdg.cnb.csic.es/pazos/mirrortree/"><i>Mirrortree</i></a> and related methodologies have been widely used to infer protein-protein interactions and functional relationships from phyogenetic trees. These methods assume that related proteins evolve in a coordinated fashion printing this signal in the topology and distances of their phylogenetic trees. Although some authors showed that co-evolution is more evident when evaluated on the surfaces/interfaces of the proteins, the lack of experimental 3D structures precludes taking advantage of this for improving the performance of the method in general.</p>

<p>To address this limitation, we benchmarked the impact of restricting <em>mirrortree</em>-like analysis to the residues predicted as accessible. In order to do that, we predicted solvent accessibility for each <em>E. coli</em> protein using multiple sequence alignments containing homologous sequences. The trees of orthologous which are the basic input of <em>mirrortree</em>-like methods where then constructed but using only the positions fulfilling certain criteria of predicted accessibility. The global performances obtained by 3 different methods (MirrorTree (MT), Profile Correlation (PC) and ContextMirror (CM)) when fed with these trees were evaluated  using 3 datasets representing different types of interactions: binary physical, co-presence in macromolecular complexes and co-membership in the same metabolic pathway.</p>	

<p>We found that the performances of the context-based methodologies (PC and CM) in predicting binary-physical interactions are increased when using trees based on predicted accessible positions. Moreover, the methods showed better performance when using alignments constructed ad-hoc for predicting accessibility (i.e. including paralogs) than when using the same alignments of orthologs used for deriving the trees.</p>

<p>The general conclusion is that predicted accessibility only helps in the prediction of protein interactions under some circumstances. The accurate context-based methods predicting binary physical interactions is the ideal case to think about excluding (predicted) not-accessible regions. Moreover, that improvement comes at no cost in terms of applicability since accessibility can be predicted for any sequence.</p> 

<p class="sectionP">More information and links</p>

<li>
David Ochoa, Ponciano Garcia-Gutierrez, David Juan, Valencia, A. and Florencio Pazos (2013).
Incorporating information on predicted solvent accessibility to the co-evolution-based study of protein interactions.
<i>Molecular bioSystems.</i>
<b>9:</b>70-76.
<br>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/23104128">[PubMed:23104128]</a>
<a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&id=23104128&retmode=ref&cmd=prlinks">[HTML]</a>
<a href="http://pubs.rsc.org/en/content/articlepdf/2013/mb/c2mb25325a">[PDF]</a>

<li><a href="http://pdg.cnb.uam.es/pazos/mirrortree/"><i>Mirrortree</i> and related programs</a>
<li><a href="http://csbg.cnb.csic.es/mtserver/"><i>Mirrortree</a></i> web server</a>


</li>



<!--FIN DEL CONTENIDO -->
<center><hr>
			<p class="footText">
				&copy; 2012, Computational Systems Biology Group. CNB-CSIC
			</p>


<script src="http://www.google-analytics.com/urchin.js" type="text/javascript">
</script>
<script type="text/javascript">
_uacct = "UA-1021511-1";
urchinTracker();
</script>
<script src="http://www.google-analytics.com/urchin.js" type="text/javascript">
</script>
<script type="text/javascript">
_uacct = "UA-1063282-1";
urchinTracker();
</script>

</body>
</html>
